app/views/step_by_step_pages/show.html.erb

```diff
@@ -15,64 +15,86 @@
-              <%= step.title %>
-              <%= render 'shared/steps/broken-links-summary', step: step %>
-            </th>
+              <th class="govuk-table__header" scope="row">
+                <%= step.title %>
+                <%= render 'shared/steps/broken-links-summary', step: step %>
+              </th>
 
-            <td class="text-right text-nowrap">
-              <%= link_to("Edit", edit_step_by_step_page_step_path(@step_by_step_page.id, step.id), class: "btn btn-primary btn-sm") %>
-              <%= link_to("Delete", step_by_step_page_step_path(@step_by_step_page.id, step.id), method: 'delete', data: { confirm: "Are you sure?" }, class: "btn btn-danger btn-sm") %>
-            </td>
+              <td class="govuk-table__cell govuk-table__cell--numeric">
+                <%= link_to("Edit", edit_step_by_step_page_step_path(@step_by_step_page.id, step.id), class: "gem-c-button govuk-button gem-c-button--inline") %>
+                <%= link_to("Delete", step_by_step_page_step_path(@step_by_step_page.id, step.id), method: 'delete', data: { confirm: "Are you sure?" }, class: "gem-c-button govuk-button govuk-button--warning gem-c-button--inline") %>
+              </td>
+            </tr>
+          <% end %>
+        <% else %>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell" colspan="3">
+              <p class="govuk-hint">No steps have been added yet.</p>
+            </tr>
           </tr>
         <% end %>
       </tbody>
     </table>
-    <div class="col-md-2 col-md-offset-10">
-      <ul class="list-unstyled step-by-step-actions text-right">
-        <li><%= link_to 'Add a new step', new_step_by_step_page_step_path(@step_by_step_page), class: "btn btn-primary btn-block step-action" %></li>
-        <li><%= link_to 'Preview', preview_url(@step_by_step_page.slug, auth_bypass_id: @step_by_step_page.auth_bypass_id), class: "btn btn-success btn-block preview-button step-action" %></li>
-      </ul>
-    </div>
-    <div>
-      <%= button_to "Check for broken links", {action: "check_links", step_by_step_page_id: @step_by_step_page.id}, class: "btn btn-primary" %>
-    </div>
+  </div>
+
+  <div class="govuk-grid-column-one-third">
+    <ul class="govuk-list step-by-step-actions__list">
+      <li>
+        <%= render "govuk_publishing_components/components/button", {
+          text: "Add a new step",
+          href: new_step_by_step_page_step_path(@step_by_step_page)
+        } %>
+      </li>
+      <li class="govuk-!-margin-top-2">
+        <%= render "govuk_publishing_components/components/button", {
+          text: "Preview",
+          href: preview_url(@step_by_step_page.slug, auth_bypass_id: @step_by_step_page.auth_bypass_id),
+          secondary: true
+        } %>
+      </li>
+      <li class="govuk-!-margin-top-3">
+        <%= link_to "Check for broken links", {action: "check_links", step_by_step_page_id: @step_by_step_page.id}, method: :post, class: "govuk-link" %>
```

Comment:
I find it odd that "Check for broken links" is link when it performs an action, and "Preview" is a button when really it's a link.